packages/react/src/__tests__/createReactClassIntegration-test.js

```diff
@@ -338,7 +338,14 @@ describe('create-react-class-integration', () => {
       root.render(<Outer />);
     });
     assertConsoleErrorDev([
-      'Component uses the legacy childContextTypes API which will soon be removed. Use React.createContext() instead.',
+      gate(flags =>
+        flags.enableOwnerStacks
+          ? [
+              'Component uses the legacy childContextTypes API which will soon be removed. Use React.createContext() instead.',
+              {withoutStack: true},
```

Comment:
this doesn't seem like it should be needed, still working on figuring out what's going on here

Reply:
think this may actually be correct. the error below for `contextTypes` has an owner stack because it is on `Foo`, which is rendered by `Outer`. `childContextTypes` is on `Outer`, however, which has no owner – it is rendered directly by `root.render`. If `Outer` is wrapped with an arbitrary wrapper component, we get a stack here.

Reply:
why wasn't this failing before though?

Reply:
Looks like a bug in the checker to me. Why do we need to gate only first message?

Reply:
> see my second comment - it seems there's no "owner" to point to (it would be the `root.render` call on L338), so there's no stack

Makes sense, but when `enableOwnerStacks` is disabled, classic component stacks should be appended, which  will be empty in this case as well, but was working well without specifying `withoutStack` property.

Reply:
that's true - it appears that with `enableOwnerStacks` disabled the stack here ends up pointing at that anonymous function. not sure if we should expect owner stacks to do the same?

Reply:
> why wasn't this failing before though?

to fully close the loop on this, @rickhanlonii, it seems its just because with `enableOwnerStacks` prior only enabled in experimental, [`disableLegacyContext` was always enabled](https://github.com/facebook/react/blame/d82a7b4e6a0b192ca222a1aa8f1f5fd6ba41fc38/packages/shared/ReactFeatureFlags.js#L192)

since this test has `// @gate !disableLegacyContext`, it plainly was never running before when `enableOwnerStacks` was gated, as there wasn't a permutation of the feature flags where `disableLegacyContext` was `false` but `enableOwnerStacks` was `true`. `enableOwnerStacks` was _only_ ever true when `disableLegacyContext` was already `true`.